images/fcos-bastion-image/root/usr/bin/fcos-download.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -x
+
+function fail() {
+  echo "$1"
+  exit 1
+}
+
+# download latest Fedora CoreOS
+mkdir -p ${TFTP_DIR}/fcos-{aarch64,x86_64}
+curl -o /tmp/stable.json https://builds.coreos.fedoraproject.org/streams/stable.json
+curl -o /tmp/fedora.gpg https://fedoraproject.org/fedora.gpg
+fcos_release=$(jq -r .architectures.aarch64.artifacts.metal.release /tmp/stable.json)
+echo -e "\n\e[31m~~ Fedora CoreOS Release ${fcos_release} ~~\e[0m\n"
```

Comment:
You sure this is visible in the journalctl logs?

Reply:
you may need to have a look at this https://serverfault.com/a/1040601 to do what you want.

Reply:
Updated to use levels <3> and <4> for colors in the journalctl logs. Verified it is working as expected.